go/util/clienttest/client_test_suite.go

```diff
@@ -23,6 +23,10 @@ type ClientTestSuite struct {
 	err        *os.File
 }
 
+type ExitError struct {
+	Code int
+}
```

Comment:
If you define an `Error()` method then callers will be able to pass `ExitError` around as an `error` not just an `interface{}`. I.e. currently:
```go
var err interface{} = ExitError{1}
_, ok := err.(error) // ok is false
```
but with an Error method:
```go
var err interface{} = ExitError{1}
_, ok := err.(error) // ok is true
```